### PR TITLE
Display status

### PR DESCRIPTION
**Pull request by heron2014**
Related to [issue 207](https://github.com/FAC-GM/app/issues/207)

@nelsonic please check this PR?

### CODE REVIEW

**Comment by nelsonic** [comment]
Code review now... :hourglass:

**Comment by nelsonic** [comment]
Looking good. 
@iteles we need to decide *when* we are going to move to following the GM wireframes, color scheme and "brand" closer.

File: test/generateStatusHelper.test.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  it('return the right status from the notes', function (done) {
+
+      //the last item of the array is the most recent
+      var notes = [
+        { status: 'interview',
+          company: 'FAC',
+          notes: 'Good candidate',
+          author: 'Anita'
+        },
+        { status: undefined,
```
Comment (iteles) [inline test/generateStatusHelper.test.js]: Whilst it's good that you've written a test for this, we shouldn't _have_ a situation where there is a status of `undefined` with a company name. We need to write an issue for this. I think it's a case of adding validation that doesn't allow someone to save the note with a company but not a status.

**Comment by iteles** [comment]
Aside from the above, please also see: https://github.com/FAC-GM/app/issues/207#issuecomment-173244135

File: test/generateStatusHelper.test.js (hunk 1 above)
Comment (SimonLab) [inline test/generateStatusHelper.test.js]: Yes I think the status should be mandatory, we need to verify this with Adam. At the status will be the latest status which is not undefined

**Comment by nelsonic** [comment]
@heron2014 just to *clarify* am I meant to review/merge #209 and *then* #213 ?
I think @iteles comment regarding not *allowing* `status=undefined` should be addressed in #209 
would you mind raising it as an `issue` so that we can *confirm* the desired behaviour with @Adam-JF  ?
thanks!

**Comment by heron2014** [comment]
@nelsonic yes exactly. First please merge [#209](https://github.com/FAC-GM/app/pull/209) and after [#213](https://github.com/FAC-GM/app/pull/213).

**Comment by nelsonic** [comment]
Merging. :+1:

**Pull request merged**